packages/api-gateway/src/drivers/http/adapters/admin_panel/handlers.js

```diff
@@ -1,11 +1,35 @@
 async function sayHello(req, reply) {
-  const result = await this.adminPanelService.sayHello();
+
+	const result = await this.adminPanelService.sayHello();
 
   return reply.code(200)
     .header('Content-Type', 'application/json; chartset:utf-8')
     .send({ result });
 }
 
+async function changeUserStatus(req, reply) {
+
+  //TODO: this is not the place for try/catch
+  try {
+    const result = await this.adminPanelService.changeUserStatus(
+      req.params.user_id,
+      req.body.status,
+      req.body.reason
+    );
+
+    return reply.code(200)
+      .header('Content-Type', 'application/json; chartset:utf-8')
+      .send(result);
+
+  }catch(e) {
+    return reply.code(400).send({
+      message: e.message
+    });
+  }
+
+}
+
 module.exports = {
-  sayHello,
-};
+	sayHello,
+  changeUserStatus
+}
```

Comment:
Can you add a semi-colon at the end of the line? it is throwing eslint warnings
```suggestion
};
```